prime-router/src/main/kotlin/fhirengine/engine/FHIRRouter.kt

```diff
@@ -451,8 +443,8 @@ class FHIRRouter(
                 )
             }
         } catch (e: SchemaException) {
-            actionLogger?.error(InvalidFilterExpressionMessage(e.message ?: ""))
-            throw e
+            actionLogger?.warn(EvaluateFilterConditionErrorMessage(e.message ?: ""))
```

Comment:
Add a default message instead of just empty string

Reply:
A default message is now provided in the error object constructor. There is no need for the UP router function to provide one.